tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -2821,6 +2819,9 @@ node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighth
 node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background.js(60267,11): error TS2304: Cannot find name 'define'.
 node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background.js(60267,32): error TS2304: Cannot find name 'define'.
 node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background.js(60268,1): error TS2304: Cannot find name 'define'.
+node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background.js(60288,5): error TS2322: Type '{ exports: { [x: string]: any; }; id: any; loaded: boolean; }' is not assignable to type '{ "../../../tests/cases/user/chrome-devtools-frontend/node_modules/chrome-devtools-frontend/front_end/audits2_worker/lighthouse/lighthouse-background": any; }'.
```

Comment:
AMD module tries to overwrite `module`, but it's now already bound and therefore gives an assignability error.